src/app/teacher/run-menu/run-menu.component.ts

```diff
@@ -8,23 +8,28 @@ import { ConfigService } from '../../services/config.service';
 import { RunSettingsDialogComponent } from '../run-settings-dialog/run-settings-dialog.component';
 import { EditRunWarningDialogComponent } from '../edit-run-warning-dialog/edit-run-warning-dialog.component';
 import { Router } from '@angular/router';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { ArchiveProjectService } from '../../services/archive-project.service';
+import { ArchiveProjectResponse } from '../../domain/archiveProjectResponse';
 
 @Component({
   selector: 'app-run-menu',
   templateUrl: './run-menu.component.html',
   styleUrls: ['./run-menu.component.scss']
 })
 export class RunMenuComponent implements OnInit {
+  private editLink: string = '';
+  protected reportProblemLink: string = '';
   @Input() run: TeacherRun;
-
-  editLink: string = '';
-  reportProblemLink: string = '';
+  @Output() runArchiveStatusChangedEvent: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Looks like we don't do anything with the emitted value, so make it void?
```suggestion
  @Output() runArchiveStatusChangedEvent: EventEmitter<void> = new EventEmitter<void>();
```